components/server/src/projects/projects-service.ts

```diff
@@ -227,9 +232,7 @@ export class ProjectsService {
 
     private async onDidCreateProject(project: Project, installer: User) {
         // Pre-fetch project details in the background -- don't await
-        /** no await */ this.getProjectOverviewCached(installer, project).catch((err) => {
-            /** ignore */
-        });
+        void this.getProjectOverview(installer, project.id);
```

Comment:
Without the catch this is a potential UnhandledPromiseRejection. Should be an eslint rule for that I guess.

Reply:
https://www.npmjs.com/package/eslint-plugin-promise, rule is 
    "promise/catch-or-return": "error",
    
We have it in the root but not the individual components, yet.

Reply:
Will be a bigger change, so fine with a follow up :wink:

Reply:
It is checked. The [void operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/void) is they way to tell eslint "this is fine".

Reply:
> "this is fine".

But I don't think it is? What if the function throws? In later nodejs versions this will trigger a process restart. Or am I missing sth? :thinking:

Reply:
>  In later nodejs versions this will trigger a process restart

do you have a reference to that?

Reply:
ah ok found it. Strange that eslint tooling is proposing this as a fix. Will update them.

Reply:
I think it will be a UnhandledPromiseRejection at runtime, which be defauols fails with exit code one ([source](https://nodejs.org/api/process.html#event-uncaughtexception)).

We could mitigate by either:
 - change the default behavior ([as we do today](https://github.com/gitpod-io/gitpod/blob/2bf3a3ad805c2e86a4494a1e8951593572ef02b6/components/server/leeway.Dockerfile#L13))
 - or installing a global handler
But either way, IMO we should avoid those, and having a concrete error message that is linked to the call site would be ideal.

Reply:
Yes, I've pushed an update

Reply:
I checked, and the diff still has a ton of `void call()` in them. What was the change?